src/kOS/Execution/ProgramContext.cs

```diff
@@ -111,13 +111,21 @@ public List<string> GetCodeFragment(int start, int stop)
         {
             var codeFragment = new List<string>();
 
-            const string FORMAT_STR = "{0,-20} {1,4}:{2,-3} {3:0000} {4} {5}";
-            codeFragment.Add(string.Format(FORMAT_STR, "File", "Line", "Col", "IP  ", "opcode", "operand"));
-            codeFragment.Add(string.Format(FORMAT_STR, "----", "----", "---", "----", "---------------------", ""));
+            const string FORMAT_STR = "{0,-20} {1,4}:{2,-3} {3:0000} {4} {5} {6}";
+            codeFragment.Add(string.Format(FORMAT_STR, "File", "Line", "Col", "IP  ", "label", "opcode", "operand"));
+            codeFragment.Add(string.Format(FORMAT_STR, "----", "----", "---", "----", "-------------------------------", "", ""));
```

Comment:
Changed the format because this isn't the first time I've had trouble diagnosing a problem because the debug dump didn't show the string label that the opcode was using.